py-polars/tests/unit/datatypes/test_temporal.py

```diff
@@ -910,22 +910,28 @@ def test_groupby_dynamic_when_conversion_crosses_dates_7274() -> None:
         )
         .with_columns(
             pl.col("timestamp")
-            .str.strptime(pl.Datetime, format="%Y-%m-%d %H:%M:%S%:z")
+            .str.strptime(pl.Datetime, format="%Y-%m-%d %H:%M:%S%:z", utc=True)
```

Comment:
passing `utc=True` and then converting to `'Africa/Lagos'` to avoid the warning

there are still tests which check the warning, but for tests where `utc` or not isn't the point of the tests, I'm just passing in `utc=True` to opt in to the new behaviour and silence the warnings